modules/loader/src/main/scala/com/snowplowanalytics/snowplow/postgres/config/ConfigPreCheck.scala

```diff
@@ -0,0 +1,30 @@
+package com.snowplowanalytics.snowplow.postgres.config
+
+import cats.effect.Sync
+import cats.data.EitherT
+import cats.implicits._
+
+import java.nio.file.{Path, Files}
+import com.snowplowanalytics.snowplow.postgres.config.LoaderConfig.Source.LocalFs
+
+import scala.util.Try
+
+/**
+  * Check if given configuration meets necessary requirements
+  */
+object ConfigPreCheck {
+  def apply[F[_]: Sync](cli: Cli[F]): EitherT[F, String, Cli[F]] =
+    EitherT(Sync[F].delay {
+      cli.config.input match {
+        // If configuration is for local source, check if given path exists
+        case LocalFs(path) =>
+          Either.fromTry(Try {
+            Files.exists(Path.of("/" + path.toString()))
+          }).flatMap { exists =>
+            if (exists) cli.asRight
+            else "Path does not exist".asLeft
```

Comment:
error message could be more explicit by something like

```scala
else s"Local source path [${Path.of("/" + path.toString())}] does not exist".asLeft
``` 

I think in this case it is important to share what we search, since current error message feels like `path` doesn't exist, whereas we search for `/ + path`